Accessibility Handbook/Book/VoiceOverGuide/AccessibilityTraits/Pages/SearchFieldTraitPage.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  SearchFieldTraitPage.swift
+//  Accessibility Handbook
+//
+//  Created by Maria Fernanda Azolin on 28/10/22.
+//
+
+import SwiftUI
+
+struct SearchFieldTraitPage: View, Page {
+    
+    let title: String = L10n.SearchFieldTrait.title
+    let animals = ["arara", "cobra", "lagarto"]
+    
+    let codeUIKit: String = """
+    myView.accessibilityTraits = [.searchField]
+    """
+    
+    let codeSwiftUI: String = """
+    .accessibilityAddTraits([.isSearchField])
+    """
+    
+    let link: String = """
+    https://developer.apple.com/documentation/uikit/uiaccessibilitytraits/1620199-searchfield
+    """
+    
+    var body: some View {
+        PageContent(next: nil) {
```

Comment:
You'll need to update with the main branch and 
```swift
PageContent(next: nil, deeplink: deeplink) {
```
We have a new deeplink structure, and each page needs to provide to the `PageContent` it's deeplink.
> That's all you ned to do, since the `Page` has a computed extension for it's deeplink